interop/src/qif.rs

```diff
@@ -365,6 +400,7 @@ impl EncodedFile {
             stream_count += 1;
         }
         fs::write(&self.file, buf)?;
+        println!("encoded in {:?}", self.file);
```

Comment:
This doesn't seem to belong?

Reply:
It actually is. When you run it, you'll want to know what happened. I figured this out writing the doc.

Reply:
Fair enough!